test/behaviour/connection/user/UserSteps.java

```diff
@@ -63,9 +63,14 @@ public void users_create(String username, String password) {
         getClient().users().create(username, password);
     }
 
-    @Then("user password: {word}, {word}")
-    public void user_password(String username, String password) {
-        getClient().users().get(username).password(password);
+    @Then("users delete: {word}")
+    public void users_delete(String username) {
+        getClient().users().delete(username);
+    }
+
+    @Then("user password update: {word}, {word}, {word}")
+    public void user_password_update(String username, String oldPassword, String newPassword) {
+        getClient().users().get(username).passwordUpdate(oldPassword, newPassword);
```

Comment:
what about one for `users_password_set`?

Reply:
Haven't done the behaviour tests yet, apologies.

Reply:
It seems as though the user behaviour steps aren't run in the current set of automation jobs in vaticle/typedb-client-java - we should decide what we want to do with these.